helper/tests/helper-tests-16/src/test/java/com/linkedin/avroutil1/compatibility/avro16/AvroCompatibilityHelperAvro16Test.java

```diff
@@ -7,7 +7,7 @@
 package com.linkedin.avroutil1.compatibility.avro16;
 
 import com.linkedin.avroutil1.Pojo;
-import com.linkedin.avroutil1.TestUtil;
+import com.linkedin.avroutil1.testcommon.TestUtil;
```

Comment:
Can you separate out the move of the class to a new package namespace in a separate PR? we can avoid ~40 files in this diff?

Reply:
will do